protocol.md

```diff
@@ -94,7 +94,7 @@ StatusObject = {
 		"staticLights": Boolean, // Static lights
 		"leftHanded": Boolean, // Left handed
 		"playerHeight": Number, // Player's height
-		"disableSFX": Boolean, // Disable sound effects
+		"sfxVolume": Number, // Disable sound effects
```

Comment:
Please add the value range if possible. Something like this (meaning values range from 0 to 1, inclusively on both ends):

```suggestion
		"sfxVolume": Number, // Disable sound effects. [0..1]
```